__tests__/app.test.js

```diff
@@ -158,6 +159,59 @@ describe("/api", () => {
               });
           });
         });
+        describe("POST", () => {
+          test("201: sends an object of the posted comment", () => {
+            return request(app)
+              .post("/api/articles/5/comments")
+              .send({ username: "butter_bridge", body: "this is a comment!" })
+              .expect(201)
+              .then(({ body }) => {
+                const { comment } = body;
+                expect(typeof comment.comment_id).toBe("number");
```

Comment:
We could be more specific with the id here, we know how many are in the database so could add 1 to that and there's our id :)